projects/opendr_ws/src/perception/scripts/object_tracking_2d_fair_mot.py

```diff
@@ -166,27 +171,53 @@ def draw_predictions(frame, predictions: TrackingAnnotation, is_centered=False,
         )
 
 
-if __name__ == "__main__":
-    # Automatically run on GPU/CPU
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--model_name", help="Name of the trained model",
+                        type=str, default="fairmot_dla34")
```

Comment:
Added choices argument, it seems redundant but it shows up when calling help and avoids users providing wrong values.

```suggestion
    parser.add_argument("-n", "--model_name", help="Name of the trained model",
                        type=str, default="fairmot_dla34", choices=["fairmot_dla34"])
```